### ISSUE TITLE
Bug report widget can not redirect if traceback is too long

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/aiidalab-widgets-base#530]
There is a limitation for the GitHub issue creating API, if the traceback is too long the button will lead to "URL is too long" 414 page, as the same error show in https://github.com/cli/cli/issues/1575

We can follow the same solution by trimming the traceback to be reported and adding a note to tell the user to copy-paste the trimmed traceback if it is useful.

**Comment by danielhollas** [comment]
Agree, although not sure from which side we should trim. I'd guess that the most useful would be to trim the middle.
